inference-engine/ie_bridges/python/install/setup.py.in

```diff
@@ -0,0 +1,287 @@
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ This conversation was marked as resolved by dkurt
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os.path
+import sys
+import errno
+import subprocess
+from pathlib import Path
+from distutils.command.install import install
+from distutils.command.build import build
+from distutils.errors import DistutilsSetupError
+from distutils.file_util import copy_file
+from setuptools import setup, find_packages, Extension
+from setuptools.command.build_ext import build_ext
+from setuptools.command.build_clib import build_clib
+
+WHEEL_LIBS_INSTALL_DIR = "openvino.libs"
+PYTHON_VERSION = f"python{sys.version_info.major}.{sys.version_info.minor}"
```

Comment:
Building Python wheel openvino
  File "/home/slyubimt/Projects/pip/openvino/build/inference-engine/ie_bridges/python/install/setup.py", line 29
    PYTHON_VERSION = f"python{sys.version_info.major}.{sys.version_info.minor}"
                                                                                                                                ^
SyntaxError: invalid syntax
inference-engine/ie_bridges/python/install/CMakeFiles/ie_wheel.dir/build.make:80: recipe for target 'ie_wheel' failed
make[2]: *** [ie_wheel] Error 1

Reply:
You use the old python version without fstrings support